packages/cli-lib/lib/watch.js

```diff
@@ -129,6 +152,7 @@ function watch(accountId, src, dest, { mode, remove, disableInitial, notify }) {
       mode,
     });
     triggerNotify(notify, 'Added', filePath, uploadPromise);
+    notifyOfThemePreview(filePath, accountId);
```

Comment:
Does this notify for every file upload? If so, I think it might be better to just show the message once

Reply:
I debounce it for that reason.  I couldn't think of a great way to know if all files have finished uploading or not.  But since it is debounced for 1 second, it should only show once in most cases

Reply:
Having just tried it on a whole folder though, it is saying the message before the uploads are all done, will look further into it

Reply:
I just stumbled on this https://github.com/SBoudrias/Inquirer.js#bottom-bar---inquireruibottombar which might be an interested way to show this

Reply:
Oh very cool.  I checked it out, but it seems to do about the same thing with our current setup.  I _think_ we'd have to swap over to their ui.log for all of the messages, which I wasn't sure if we'd want to do since we have our own logger already.

Either way I realized I was being dumb and can check queue.size and make sure it is 0 before displaying a message.  That + the debounce seems to work great